### ISSUE TITLE
How to select checkpoint when fine-tuning on MSRVTT

### ISSUE THREAD

**Issue by chaochen99** [m-bain/frozen-in-time#49]
Dear Sir, 

During fine-tuning of MSRVTT, we found no validation set in the code. How to choose a checkpoint?

Looking forward to your reply.

Thanks！

**Comment by m-bain** [comment]
Hi,
correct, unfortunately for msr-vtt people use validation set and test set the same. You can just train for 50 epochs or so which is when train/val converges I believe. Best SOTA numbers by other works likely use val set performance to pick checkpoint.
